features/step_definitions/edit_page_steps.rb

```diff
@@ -142,3 +163,18 @@
 Then(/^no email notifications are sent$/) do
   expect(ActionMailer::Base.deliveries).to be_empty
 end
+
+Then(/^I should see "(.*?)" in the list of linkable pages/) do |name|
+  @linkable_page = edit_page.link_manager.pages.results.find do |row|
+    row.page_name.text.include? name
+  end
+  expect(@linkable_page).to_not be_nil
+end
+
+Then(/^the "([a-zA-Z]{2})" and "([a-zA-Z]{2})" versions are available for linking$/) do |*locales|
+  locales.each do |locale|
+    expect(@linkable_page.send(locale)).
```

Comment:
This is a tricky one. Not having steps dependant is good design, but in this case it gets awkward if we want to bundle both tests together into `I should see "(.*?)" in the list of linkable pages` and `the "([a-zA-Z]{2})" and "([a-zA-Z]{2})" versions are available for linking` steps. The step wording gets awkward, etc. And it's nice to test this criteria separately instead of squishing them into the same step.